jupyterhub/handlers/base.py

```diff
@@ -22,6 +23,7 @@
 from ..objects import Server
 from ..spawner import LocalProcessSpawner
 from ..utils import url_path_join
+from ..metrics import SPAWN_DURATION_SECONDS, PROXY_ADD_DURATION_SECONDS
```

Comment:
Do you think `ADD_PROXY_DURATION_SECONDS` is clearer to indicate that it's how long adding a proxy takes?

Reply:
I think general prometheus guidelines are to do things like PROXY_ADD_DURATION_SECONDS and PROXY_REMOVE_DURATION_SECONDS, so they sort together. I can change it if you feel strongly, but would like to keep it as is!

Reply:
Let's leave as is but clarify in a doc string or comment since I initially interpreted as adding seconds vs. adding user to the proxy.

Reply:
I added more docs on metrics.py, and renamed the spawn metric so it now follows a '<noun>_<verb>_<type-suffix>' pattern. Great catch on confusion!